website/docs/reference/resource-configs/grants.md

```diff
@@ -249,7 +249,39 @@ models:
 
 <div warehouse="Redshift">
 
-* Granting to / revoking from is only fully supported for Redshift users (not groups or roles).
+For Redshift, you need to distinguish between users, roles, and groups. It is important to explicitly grant roles and groups access to your models. You can revoke access as well.
+
+**For users:**
+When granting access to individual users, use the `+grants` syntax in your configuration to add or modify permissions for these users without replacing existing configurations. In the following example, you only need to list the users:
+
+```yaml
+models:
+  my_schema:
+    schema: my_schema
+    description: "My schema"
+    +grants:
+      select: ["gspider"]  # example of user access
+```
+
+**For roles and groups:**
+Inheritance of grants are not supported for groups and roles in Redshift. Accordingly, for every model, you need to provide the complete list of roles or groups that require access each time you make a change to a model.
+
+Additionally, must prefix each role or group with 'role' or 'group':
+
+```yaml
+models:
+  my_schema:
+    schema: my_schema
+    description: "Schema for reporting"
+    grants:
+      select: ["role my_role", "group my_group"]
+```
+
+Some things to note:
+
+- Ensure that any roles or groups mentioned must already exist in Redshift.
+- Use the keyword `+grants` to add to or modify users for existing grants (rather than replace), however this syntax does not apply to roles and groups.
```

Comment:
I still don't think that the `+grants` does anything different than `grants`. The important caveat is really my edit. 

```suggestion
- Adding or modifying users for existing grants (rather than replace), is not supported for roles and groups.
```

Reply:
edit -- I'm totally wrong here after reading the Notes section at the end of the 
[grants: Grant config inheritance](https://docs.getdbt.com/reference/resource-configs/grants#grant-config-inheritance)

Now, I think this is the right language.

```suggestion
- The merge strategy of grants (via the `+` prefix in front of `+grants`) is not supported for roles and groups.
```